acestep/training/dataset_builder_modules/audio_io.py

```
@@ -29,6 +30,36 @@ def load_caption_file(audio_path: str) -> Tuple[str, bool]:
+    """
+    base_path = os.path.splitext(audio_path)[0]
+    json_path = base_path + ".json"
+    if not os.path.exists(json_path):
+        return {}, False
+    try:
+        with open(json_path, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        if isinstance(data, dict):
+            logger.debug(f"Loaded JSON metadata from {json_path}")
+            return data, True
+        return {}, False
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)